aptos-move/block-executor/src/captured_reads.rs

```diff
@@ -913,6 +913,10 @@ mod test {
         type Key = KeyType<u32>;
         type Tag = u32;
         type Value = ValueType;
+
+        fn user_txn_bytes_len(&self) -> usize {
+            0
```

Comment:
nit: if you define this on the trait, you won't need to redefine it everywhere, right?

Reply:
I put it this way because I feel safer to see 0 only in test code, instead of accidentally having 0 for prod :)

but not a strong preference, let me know what do you think